spec/support/shared_validator_specs.rb

```diff
@@ -124,6 +128,17 @@ def alphanumeric_string(length)
   end
 end
 
+shared_examples "a nilable guid validator" do
```

Comment:
Can you please (in a second commit) use this for [photo_validator_spec.rb](https://github.com/diaspora/diaspora_federation/blob/e26a86c173a7fdd222bd3cba677f03cf08c6487c/spec/lib/diaspora_federation/validators/photo_validator_spec.rb#L19) too?